api/hasura/actions/_handlers/createVaultTx.ts

```diff
@@ -0,0 +1,69 @@
+import assert from 'assert';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import {
+  ValueTypes,
+  vault_tx_types_enum,
+} from '../../../../api-lib/gql/__generated__/zeus';
+import { adminClient } from '../../../../api-lib/gql/adminClient';
+import { verifyHasuraRequestMiddleware } from '../../../../api-lib/validate';
+import {
+  VaultLogInputSchema,
+  VaultLogUnionSchema,
+  composeHasuraActionRequestBodyWithSession,
+  HasuraUserSessionVariables,
+} from '../../../../src/lib/zod';
+
+async function handler(req: VercelRequest, res: VercelResponse) {
+  const {
+    session_variables: { hasuraProfileId },
+    input: { payload },
+  } = composeHasuraActionRequestBodyWithSession(
+    VaultLogInputSchema,
```

Comment:
seems to be missing validation whether `circle_id`,`vault_id` is under the `organisation`